setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+with open('README.md', encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='bitalg',
+    description='Laboratory classes in the Geometric Algorithms course for Computer Science students at the AGH University.',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    author='AGH BIT Student Scientific Group',
+    packages=['bitalg'],
+    python_requires='>=3.11'
```

Comment:
Requiring users to have a version >=3.11 is too intrusive, consider changing to >=3.8

Reply:
From the very beginning we required Python version 3.11. https://github.com/aghbit/Algorytmy-Geometryczne/blob/master/alg-geo.txt#L70C1-L70C76

Reply:
We proposed an environment with version 3.11, we did not require it